lib/babe/secondary.go

```diff
@@ -68,25 +68,25 @@ func verifySecondarySlotVRF(digest *types.BabeSecondaryVRFPreDigest,
 	numAuths int,
 	randomness Randomness,
 ) (bool, error) {
-	expected, err := getSecondarySlotAuthor(digest.SlotNumber(), numAuths, randomness)
+	expected, err := getSecondarySlotAuthor(digest.SlotNumber, numAuths, randomness)
 	if err != nil {
 		return false, err
 	}
 
 	logger.Trace("verifySecondarySlotPlain",
-		"index", digest.AuthorityIndex(),
+		"index", digest.AuthorityIndex,
 		"pub", pk.Hex(),
 		"number of authorities", numAuths,
-		"slot", digest.SlotNumber(),
+		"slot", digest.SlotNumber,
 		"epoch", epoch,
 		"randomness", randomness,
 		"expected index", expected,
 	)
 
-	if digest.AuthorityIndex() != expected {
+	if digest.GetAuthorityIndex() != expected {
```

Comment:
are the getters needed, if the fields are all exported now? ie. you can just do `digest.AuthorityIndex`

Reply:
I don't think they are needed most of the time, but some instances use the BabePreRuntimeDigest interface rather than the specific struct and then I think we need the getters. If there is a way to change that though that I am missing I would be happy (and prefer lol) to remove the getters